ArdentHQ/arkscan
# feat: arkconnect navbar button

## Summary

https://app.clickup.com/t/86drwqqf8

requires https://github.com/ArkEcosystem/laravel-foundation/pull/613

Todo:

- [x] Mobile state - awaiting designs
- [x] .env option to disable & add a test for it

To test:

- add `ARKCONNECT_ENABLED=true` to `.env`
- make sure you're connecting to a `https` instance of explorer. For valet, you can use `valet secure` to force https - use `valet unsecure` to change it back to `http`







## Checklist
